diagnostic.md

```diff
@@ -150,7 +174,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will be to to the right at the top of the page, because it floats right, the rest will stack on top of each other to the left, except for .divFour, which will skip to the next line, because it clears left and noq does not allow anything to float on its left.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).